src/gdcdictionary/schemas/_terms_enum.yaml

```diff
@@ -90218,6 +90302,20 @@ stage_i:
         term_url: "https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C112007"
         term_id: C112007
         term_version: 20.05a
+    ensat_pathologic_stage:
+      description: >-
+        Stage I includes: T1, N0, M0. Tumor 5 cm or less in greatest dimension. The
+        tumor has not invaded the surrounding tissues or organs and has not spread
```

Comment:
The `ensat_pathologic_stage` descriptions are missing the reference (ENSAT 7th Ed, 2009). Not sure if this is absolutely necessary.